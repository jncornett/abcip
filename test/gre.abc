d ( stack="eth:ip4:gre:ip4:gre:ip6:tcp" )
c ( 3:a="172.16.32.64", ip6:a="fc00:1:2::a:b" )
a ( syn, 4:seq=1,sre="|0 0 0 0|" )

