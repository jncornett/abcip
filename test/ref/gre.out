IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto GRE (47), length 120)
    172.16.32.64 > 10.9.8.7: GREv0, Flags [routing present, sequence# present], sum 0x0, off 0x0, seq 1, length 100
	IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto GRE (47), length 84)
    10.1.2.5 > 10.9.8.7: GREv0, Flags [none], length 64
	IP6 (flowlabel 0x00001, hlim 64, next-header TCP (6) payload length: 20) fc00:1:2::a:b.48620 > ::ffff:10.9.8.7.8: Flags [S], cksum 0xc3c4 (correct), seq 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0078 0001 0000 402f 9bf6 ac10 2040 0a09  .x....@/.....@..
	0x0020:  0807 5000 0800 0000 0000 0000 0001 0000  ..P.............
	0x0030:  0000 4500 0054 0001 0000 402f 5c65 0a01  ..E..T....@/\e..
	0x0040:  0205 0a09 0807 0000 86dd 6000 0001 0014  ..........`.....
	0x0050:  0640 fc00 0001 0002 0000 0000 0000 000a  .@..............
	0x0060:  000b 0000 0000 0000 0000 0000 ffff 0a09  ................
	0x0070:  0807 bdec 0008 0000 0001 0000 0000 5002  ..............P.
	0x0080:  2000 c3c4 0000                           ......
